Source/RomanNumeral.h

```diff
@@ -12,6 +12,29 @@
  * @param input The number to convert.
  * @param output String buffer in which to store the converted Roman numeral.
  */
```

Comment:
This doc string belongs to `RomanNumeral_Convert` so make sure it stays right above it